.github/workflows/external.yml

```diff
@@ -148,6 +148,30 @@ jobs:
       - name: cargo clippy
         run: cargo xclippy -D warnings
 
+  move-clippy:
+    needs: diff
+    if: needs.diff.outputs.isRust == 'true'
```

Comment:
should this be `isMove`?

Reply:
or is this meaning ".move" code vs code in `external/move`?

Reply:
Because this is meant to run on `external-crates/move` I called it `move-clippy` -- happy to rename it.